### ISSUE TITLE
Input activation format

### ISSUE THREAD

**Issue by ybzhou** [HawkAaron/warp-transducer#9]
I would like to use the pytorch binding for transducer loss. In the comments it is shown

`acts: Tensor of (batch x seqLength x labelLength x outputDim) containing output from network`

However, it is not clear if the input activation needs to be normalized probabilities (i.e. after softmax), log probability (i.e. log softmax) or unnormalized values. Could you please clarify on this? 

Thank you very much.

**Comment by HawkAaron** [comment]
Hi, 

Sorry for that, I have given a simple example: https://github.com/HawkAaron/warp-transducer/blob/master/pytorch_binding/README.md

For GPU version, just pass the activations without softmax, for the CPU version, you need to call `log_softmax` explicitly.

**Comment by ybzhou** [comment]
Thanks !
